astropy/visualization/wcsaxes/tests/datasets.py

```diff
@@ -4,6 +4,7 @@
 
 import time
 
+from ...extern.six.moves import urllib
```

Comment:
Oh yes, silly me, I pushed that in a hurry before leaving and I didn't check. Thanks for noticing this !